provider/machinepool/hcp/machine_pool_resource.go

```diff
@@ -180,6 +180,10 @@ func (r *HcpMachinePoolResource) Schema(ctx context.Context, req resource.Schema
 				ElementType: types.StringType,
 				Optional:    true,
 			},
+			"kubelet_configs": schema.StringAttribute{
+				Description: "Name of the kubelet config applied to the machine pool. A single kubelet config is allowed. Kubelet config must already exist.",
```

Comment:
Maybe reword it to " A single kubelet config can be applied at a time" instead of "allowed" since HCP clusters can hold more than one.